### PR TITLE
[KOA-5756] Add stories for colour tokens

### PR DESCRIPTION
**Pull request by anambl**
Add Storybook stories for all colour tokens - these will be used when running the project locally to visualise any changes to tokens.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -3,9 +3,8 @@ module.exports = {
     "../stories/(tokens|mixins)/**/*.stories.@(js|jsx|ts|tsx)",    
   ],
   "addons": [
-    "@storybook/addon-links",
```
Comment (anambl) [inline .storybook/main.js:6]: no need for all these addons at this point

File: stories/tokens/colours/colours.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  colourSwatch.style.backgroundColor = backgroundColor;
+
+  const colourName = document.createElement('div');
+  colourName.className = 'bpk-colour-name';
+  colourName.innerText = name;
+
+  colourContainer.appendChild(colourSwatch);
+  colourContainer.appendChild(colourName);
+
+  return colourContainer;
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.js:38]: Could we not do?

```suggestion
  return (
    <div className='bpk-color-container'>
       <div className='bpk-color-swatch' style={{backgroundColor: backgroundColor}} />
       <div className='bpk-color-name'> {name} </div>
     </div>
  )
```

File: stories/tokens/colours/colours.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../packages/bpk-mixins/index.scss';
+
+.bpk-colour-container {
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.scss:21]: Code should use American `color`

```suggestion
.bpk-color-container {
```

File: stories/tokens/colours/colours.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  padding: bpk-spacing-base();
+  align-items: center;
+  border-radius: $bpk-border-radius-md;
+  background-color: $bpk-surface-default-day;
+  box-sizing: border-box;
+
+  @include bpk-border-sm($bpk-line-day);
+}
+
+.bpk-colour-swatch {
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.scss:34]: Code should use American `color`
```suggestion
.bpk-color-swatch {
```

File: stories/tokens/colours/colours.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  height: bpk-spacing-md() * 15;
+  border-radius: $bpk-border-radius-sm;
+
+  @include bpk-border-sm($bpk-line-day);
+}
+
+.bpk-colour-name {
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.scss:41]: Code should use American `color`
```suggestion
.bpk-color-name {
```

File: stories/tokens/colours/colours.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+};
+
+export default createColourContainer;
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.js:41]:
```suggestion
export default createColorContainer;
```

File: stories/tokens/colours/colours.stories.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+export const SurfaceColour = Template.bind({});
+SurfaceColour.args = {
+  colourGroup: surfaceColors,
+};
+
+export const TextColour = Template.bind({});
+TextColour.args = {
+  colourGroup: textColors,
+};
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.stories.js:77]: Code should use American `color`

File: stories/tokens/colours/colours.stories.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+  const storyWrapper = document.createElement('div'); // eslint-disable-line no-undef
+  storyWrapper.style.display = 'flex';
+  storyWrapper.style.flexWrap = 'wrap';
+
+  Object.keys(colourGroup).forEach((colour) => {
+    const colourContainer = createColourContainer(colourGroup[colour], colour);
+    storyWrapper.appendChild(colourContainer);
+    return storyWrapper;
+  });
+  return storyWrapper;
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.stories.js:46]: Can we not use React for this?

**Review by olliecurtis — CHANGES_REQUESTED** [review]

File: stories/tokens/colours/colours.stories.js [hunk 9]
```diff
@@ -0,0 +1,77 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.stories.js:4]:
```suggestion
 * Copyright 2016 Skyscanner Ltd
```

File: stories/tokens/colours/colours.js [hunk 10]
```diff
@@ -0,0 +1,41 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.js:4]:
```suggestion
 * Copyright 2016 Skyscanner Ltd
```

File: stories/tokens/colours/colours.scss [hunk 11]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
```
Comment (olliecurtis) [inline stories/tokens/colours/colours.scss:4]:
```suggestion
 * Copyright 2016 Skyscanner Ltd
```

**Review by olliecurtis — CHANGES_REQUESTED** [review]

File: stories/tokens/colours/colours.stories.js (hunk 8 above)
Comment (anambl) [reply stories/tokens/colours/colours.stories.js:46]: if we use React, we need to add it as a dependency meaning one more thing we need to maintain and one more repo to add to the list of repos in which we need to do the react version update

File: stories/tokens/colours/colours.js (hunk 2 above)
Comment (anambl) [reply stories/tokens/colours/colours.js:38]: nope, not unless we add React as a dependency

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**